CHANGELOG.unreleased.md

```diff
@@ -0,0 +1,24 @@
+# Unreleased changes
+
+Add here any changes made in a PR that are relevant to end users. Allowed sections:
+
+* Fixed
+* Changed
+* Added
+* Deprecated
+
+Section headings should be at level 3 (e.g. `### Added`).
+
+## Unreleased
```

Comment:
i think you are missing this change: https://github.com/wandb/wandb/commit/6f9d242e4d445cd668f6e3201f7457b395c39181#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR26

Reply:
updated!